site/en/docs/devtools/javascript/reference/index.md

```diff
@@ -275,6 +303,51 @@ inputsAreEmpty (get-started.js:22)
 onClick (get-started.js:15)
 ```
 
+## Navigate the file tree {: #file-tree}
+
+Use [the **Page** pane](/docs/devtools/javascript/sources/#files) to navigate the file tree.
+
+### Group authored and deployed files in the file tree {: #group-authored-and-deployed }
+
+{% Aside %}
+**Note**: This is a {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/XfSWf04g2cwpnFcmp40m.svg", alt="Experimental.", width="20", height="20" %} preview feature available from Chrome version 104.
+{% endAside %}
+
+When developing web applications using frameworks (for example, [React](https://reactjs.org/) or [Angular](https://angular.io/)), it can be difficult to navigate sources due to the minified files generated by the build tools (for example, [Webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/)).
```

Comment:
```suggestion
When developing web applications using frameworks (for example, [React](https://reactjs.org/) or [Angular](https://angular.io/)), it can be difficult to navigate sources due to the minified files generated by the build tools (for example, [webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/)).
```